lib/elixir/lib/file.ex

```diff
@@ -1258,6 +1258,11 @@ defmodule File do
   One may also consider passing the `:delayed_write` option if the stream
   is meant to be written to under a tight loop.
 
+  ## Byte order marks
+
+  If you pass `:skip_bom` in the modes parameter, the stream will
```

Comment:
Agreed with @lexmag, it feels slightly inconsistent now that we removed all the "strip" stuff in `String` 😕